server/helpers/file.coffee

```diff
@@ -13,51 +11,6 @@ module.exports =
             path = "/#{path}"
         path
 
-    # Put right headers in response, then stream file to the response.
-    processAttachment: (req, res, next, download) ->
-        file = req.file
-
-        # Configure headers
-        encodedFileName = encodeURIComponent file.name
-        if download
-            contentHeader = "attachment; " + \
-                            "filename*=UTF8''#{encodedFileName}"
-        else
-            contentHeader = "inline; " + \
-                            "filename*=UTF8''#{encodedFileName}"
-        res.setHeader 'content-disposition', contentHeader
-
-        # Perform download with the lowel level node js api to avoid too much
-        # memory consumption.
-        url = "/data/#{file.id}/binaries/file"
-        requester = downloader.download url, (stream) ->
-            if stream.statusCode is 200
-                stream.pipefilter = (source, dest) ->
-                    XSSmimeTypes = ['text/html', 'image/svg+xml']
-                    if source.headers['content-type'] in XSSmimeTypes
-                        dest.setHeader 'content-type', 'text/plain'
-
-                # Set headers from data system response data.
-                res.setHeader 'content-length', stream.headers['content-length']
-                res.setHeader 'content-type', stream.headers['content-type']
```

Comment:
https://github.com/cozy/cozydb/blob/master/src/controller.coffee#L117-L127
It actually copy all headers from couchdb to the client